android/anc/src/main/java/org/smartregister/fhircore/anc/ui/login/AncLoginService.kt

```diff
@@ -19,16 +19,49 @@ package org.smartregister.fhircore.anc.ui.login
 import android.content.Intent
 import javax.inject.Inject
 import org.smartregister.fhircore.anc.ui.family.register.FamilyRegisterActivity
+import org.smartregister.fhircore.anc.ui.otp.OtpLoginActivity
+import org.smartregister.fhircore.anc.ui.otp.OtpSetupActivity
 import org.smartregister.fhircore.engine.ui.login.LoginActivity
 import org.smartregister.fhircore.engine.ui.login.LoginService
+import org.smartregister.fhircore.engine.util.FORCE_LOGIN_VIA_USERNAME
+import org.smartregister.fhircore.engine.util.SharedPreferencesHelper
 
-class AncLoginService @Inject constructor() : LoginService {
+class AncLoginService @Inject constructor(val sharedPreferencesHelper: SharedPreferencesHelper) :
```

Comment:
I'm unclear on why any of this is in ANC, don't we want to make login generic?

Reply:
fixed, will pushed with other changes pending

Reply:
pushed